examples/ember-cli/package.json

```diff
@@ -6,7 +6,8 @@
     "build": "ember build",
     "build-storybook": "yarn build && cp -r public/* dist && build-storybook -s dist",
     "dev": "ember serve",
-    "storybook": "yarn build && start-storybook -p 9009 -s dist, public"
+    "storybook": "yarn build && start-storybook -p 9009 -s dist, public",
```

Comment:
I don't know if that is the cases, I know I needed to copy the files for build-storybook, these are needed

Reply:
When I added a new file `logo1.png` into `public` it automatically gets copied into `dist` when I `ember build`.

When I `yarn storybook` and try to access it at the following URL, it succeeds:

http://localhost:9009/logo1.png

When I remove the file from `dist` and try to access it at the following URL, it 404s:

http://localhost:9009/logo1.png

Therefore, I believe:
1. The syntax "dist, public" doesn't work (maybe no comma is needed?), or possibly Storybook's ability to serve from multiple directories doesn't work.
2. We don't even need to be serving from `public` because `ember build` does the copying even when I don't use the file anywhere.

Reply:
makes sense, tested this also and it works.